### PR TITLE
models for 2 jacos with collision geom. in each link, a torso, and a large box

### PR DESCRIPTION
**Pull request by JoseBarreiros-TRI**

### CODE REVIEW

File: models/box_30_40_40cm.sdf [hunk 1]
```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0"?>
+<sdf version="1.7">
+  <model name="box_to_push">
```
Comment (aykut-tri) [inline models/box_30_40_40cm.sdf:3]: Consider updating the model name

File: models/box_30_40_40cm.sdf [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    <link name="box">
+      <inertial>
+        <mass>5.0</mass>
+        <inertia>
+          <ixx>0.00375</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>0.00375</iyy>
+          <iyz>0</iyz>
+          <izz>0.00375</izz>
```
Comment (aykut-tri) [inline models/box_30_40_40cm.sdf:14]: Please use the actual inertia values and update the scaling if necessary. For m=5.0 kg, lx=0.3 m, ly=0.4 m, lz=0.4 m, it should be:
Ixx: 0.13333333333333336
Iyy: 0.10416666666666666
Izz: 0.10416666666666666

File: models/j2s7s300_arm_sphere_link_collisions_v2.sdf [hunk 3]
```diff
@@ -0,0 +1,671 @@
+<?xml version="1.0"?>
```
Comment (aykut-tri) [inline models/j2s7s300_arm_sphere_link_collisions_v2.sdf:1]: Sorry for being annoying :grimacing: but do you mind:
1. adding new lines before and after the added collision geoms and 
2. sorting added spheres one-by-one (e.g., visual1, collision, visual2, collision2)  manner
for better readability. Feel free to ignore the second but I think the first one could really help, at least it'd help me.

**Review by aykut-tri — COMMENTED** [review]
Looks good, just some easy requests.

File: models/box_30_40_40cm.sdf (hunk 2 above)
Comment (JoseBarreiros-TRI) [reply models/box_30_40_40cm.sdf]: good catch. done

File: models/box_30_40_40cm.sdf (hunk 1 above)
Comment (JoseBarreiros-TRI) [reply models/box_30_40_40cm.sdf]: done

**Comment by JoseBarreiros-TRI** [comment]
@aykut-tri thanks for the reviews. It's ready to merge.

**Review by aykut-tri — APPROVED** [review]
Nice, LGTM

**Pull request merged**